.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+style = defaultWithAlign
```

Comment:
Indeed, ignoring whitespace is support not on in git blame, but in (afaik) all git functions. And it's supported by GitHub's UI.

I feel _strongly_ that vertically aligned code is _much_ more legible. So I'm against this decision.

But I've dropped it and updated the PR nonetheless.